# Generated by Django 2.1.1 on 2019-08-01 01:32

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('parqueaderos', '0024_auto_20190730_1942'),
    ]

    operations = [
        migrations.CreateModel(
            name='Factura',
            fields=[
                ('serial', models.CharField(max_length=25, primary_key=True, serialize=False, unique=True, validators=[django.core.validators.MinLengthValidator(5), django.core.validators.MaxLengthValidator(25)])),
                ('name', models.CharField(max_length=25, unique=True, validators=[django.core.validators.MinLengthValidator(5), django.core.validators.MaxLengthValidator(25)])),
                ('phone', models.PositiveIntegerField(validators=[django.core.validators.MaxValueValidator(9999999999)])),
                ('ubication', models.CharField(max_length=25, validators=[django.core.validators.MinLengthValidator(5), django.core.validators.MaxLengthValidator(25)])),
                ('id_user', models.CharField(max_length=20)),
                ('placa', models.CharField(max_length=6)),
                ('in_date', models.DateTimeField()),
                ('out_date', models.DateTimeField(auto_now_add=True)),
                ('total', models.PositiveIntegerField(validators=[django.core.validators.MaxValueValidator(9999999999)])),
            ],
        ),
    ]
